subgraphs/uniswap-forks/src/common/creators.ts

```diff
@@ -251,6 +251,21 @@ export function createSwapHandleVolumeAndFees(
   const token0 = getOrCreateToken(pool.inputTokens[0]);
   const token1 = getOrCreateToken(pool.inputTokens[1]);
 
+  if (amount0In == BIGINT_ZERO && amount1In == BIGINT_ZERO) {
```

Comment:
Hmm, I think it should be `!=` ? So if both amounts are != 0 then we ignore the swap